sig-and-wg/sig-core/proposals/test-folder-consolidation.md

```diff
@@ -0,0 +1,67 @@
+# Test folder consolidation
+
+## Introduction
+The [tests](https://github.com/kyma-project/kyma/tree/master/tests) folder of the kyma-project contains all tests executed against a running kyma cluster to assure integrity and functional correctness of the cluster when all modules are installed together.
+In kyma, that kind of tests are called _acceptance tests_.
+All subfolders in the _tests_ directory define one test suite, usually focussing on one component.
```

Comment:
```suggestion
All subfolders in the `tests` directory belong to one test suite, usually focusing on one component.
```

Reply:
One folder means one test suite, so they are really defining it
```suggestion
All subfolders in the `tests` directory define one test suite, usually focusing on one component.
```